docs/user-guide.md

```diff
@@ -4,6 +4,21 @@ To best understand the benefits of Akri and jump into using it, we recommend you
 ## Getting Started
 To get started using Akri, you must first decide what you want to discover and whether Akri current supports a protocol that can be used to discover resources of that type. To see the list of currently supported protocols, see our [roadmap](./roadmap.md).
 
+### Understanding Akri Helm charts
+Akri is most easily deployed with Helm charts.  Helm charts provide convenient packaging and configuration.
+
+Starting in v0.0.36, there will be a Helm chart published in **akri-dev** for each build version.  Each Akri build is verifeid with end-to-end tests on Kuberenetes, K3s, and MicroK8s, but these are likely to be less stable than our Releases.  You can deploy these versions of Akri with this command (note: **akri-dev**):
```

Comment:
"verified" and "Kubernetes".
Also, this is a great [VSCode extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) for catching spelling errors!

Reply:
nice!